docs: link to GetEnvoy.io for pre-built binaries (#7814)
envoyproxy/envoy: docs/root/install/building.rst

  organizations track and deploy master in production. We encourage you to do the same so that
  issues can be reported as early as possible in the development process.

> There are packaged Envoy pre-built binaries in rpm, deb, and Homebrew packages are available via

Reviewer: ```suggestion
Packaged Envoy pre-built binaries for a variety of platforms are available via
```

`GetEnvoy.io <https://www.getenvoy.io/>`_.

We will consider producing additional binary types depending on community interest in helping with

> CI, packaging, etc. Please open an `issue <https://github.com/envoyproxy/envoy/issues>`_ in GitHub

Reviewer: Maybe add a GH link to getenvoy here and tell people to open issue against that repo for prebuilt binaries for different platforms? WDYT?

Author: SGTM, done
